model/host/hostutil.go

```diff
@@ -249,7 +249,7 @@ func (h *Host) ForceReinstallJasperCommand(settings *evergreen.Settings) string
 	if settings.Splunk.Populated() {
 		params = append(params,
 			fmt.Sprintf("--splunk_url=%s", settings.Splunk.ServerURL),
-			fmt.Sprintf("--splunk_token=%s", settings.Splunk.Token),
+			fmt.Sprintf("--splunk_token_path=%s", h.splunkTokenFilePath()),
```

Comment:
should we support both?

Reply:
The CLI still supports both, but the splunk URL will be visible from `ps` output if you don't use a file.